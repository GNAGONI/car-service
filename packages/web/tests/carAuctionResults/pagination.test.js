import React from 'react'
import renderer from 'react-test-renderer'
import PaginationSection from '../../components/carAuctionResults/paginationSection'

it('renders correctly', () => {
  const tree = renderer.create(<PaginationSection auctionsPerPageOptions={[]} />).toJSON()
  expect(tree).toMatchSnapshot()
})
