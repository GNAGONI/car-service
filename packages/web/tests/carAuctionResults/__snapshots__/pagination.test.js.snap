// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  aria-label="Page navigation example"
  className="nav-paging"
>
  <form
    className="show-form form-inline"
  >
    <label>
      Show:
    </label>
    <div
      className="form-group"
    >
      <div
        className="css-btephf"
        onKeyDown={[Function]}
      >
        <div
          className="css-55nwcx"
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <div
            className="css-cys4jg"
          >
            <div
              className="css-xp4uvy"
            />
            <input
              className="css-14uuagi"
              disabled={false}
              id="react-select-2-input"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              readOnly={true}
              tabIndex="0"
              value=""
            />
          </div>
          <div
            className="css-1wy0on6"
          >
            <span
              className="css-rb20kt"
            />
            <div
              aria-hidden="true"
              className="css-1ep9fjw"
              onMouseDown={[Function]}
              onTouchEnd={[Function]}
            >
              <svg
                aria-hidden="true"
                className="css-19bqh2r"
                focusable="false"
                height={20}
                viewBox="0 0 20 20"
                width={20}
              >
                <path
                  d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
  <ul
    className="pagination"
  >
    <li
      className="previous disabled"
    >
      <a
        aria-disabled="true"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        <i
          className="icon-arrow-left"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
    </li>
    <li
      className="page-item active"
    >
      <a
        aria-current="page"
        aria-label="Page 1 is your current page"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        1
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 2"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        2
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 3"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        3
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 4"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        4
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 5"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        5
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 6"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        6
      </a>
    </li>
    <li
      className="break"
    >
      <a
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        ...
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        aria-current={null}
        aria-label="Page 10"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        10
      </a>
    </li>
    <li
      className="next"
    >
      <a
        aria-disabled="false"
        className="page-link"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex="0"
      >
        <i
          className="icon-arrow-right"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </li>
  </ul>
</nav>
`;
